src/api/EmployerFavouritesApi/Controllers/ApprenticeshipsController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using DfE.EmployerFavourites.ApplicationServices.Commands;
+using DfE.EmployerFavourites.Web.Security;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace DfE.EmployerFavourites.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ApprenticeshipsController : ControllerBase
+    {
+        private readonly ILogger<ApprenticeshipsController> _logger;
+        private readonly IMediator _mediator;
+
+        public ApprenticeshipsController(ILogger<ApprenticeshipsController> logger, IMediator mediator)
+        {
+            _logger = logger;
+            _mediator = mediator;
+        }
+
+        // GET api/values
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<string>>> Get(string employerAccountId)
+        {
+            try
+            {
+
+                var apprenticeships = await _mediator.Send(new GetApprenticeshipFavouritesRequest() {EmployerAccountID = employerAccountId});
+
+                return Ok(apprenticeships);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                return BadRequest();
```

Comment:
Is this a correct status code for all exceptions? This implies the error is always with caller and how they've made the request. 
We throw an `ArgumentException` in the handler if the accountId is invalid. Should this return a 400 and anything else return a 500?